static/css/main.css

```diff
@@ -521,4 +513,4 @@ code:not(pre *)  {
     font-size: 1em;
     background-color: rgb(239, 241, 243);
     font-size: 95%;
-}
\ No newline at end of file
+}
```

Comment:
Another thing that's missing here is an updated style conditional for any code block. Here's how dark mode renders code:
![image](https://user-images.githubusercontent.com/6397962/184507774-84cc77f8-74b4-4d29-8cb5-617f929c5ed1.png)

Vs standard light mode:
![image](https://user-images.githubusercontent.com/6397962/184507782-ab59742f-47f3-4b15-962b-7408929466f5.png)

In dark mode, it sets
```css
html.dark body {
  color: #ddd;
}
```
Inside a `<code>` block, which it probably shouldn't do (assuming we want to keep white backgrounds for code blocks in dark mode)

Reply:
The code block is just inheriting the color because nothing else overrides it. Easiest way I can see to change this behavior is to change dark.css line 69 (`html.dark .highlight`) to include `color: #333;` which just makes it so it never changes color when the mode changes.

Reply:
Actually this seems a bit more in-depth than that. The chroma library seems to need some attention to make highlighted code blocks to make sure that we don't override it unnecessarily and it still displays properly.

I suggest addressing the other issues in the PR first, closing it, and then I can work on the CSS to make sure it works properly across themes.

Reply:
I think we need to address the code highlighting in this PR, since otherwise the dark mode (once enabled) renders code blocks unreadable. 

This should work:
```css
html.dark td pre code {
  color: #333;
}
html.dark code:not(pre *)  {
  color: #ddd;
  background-color: #333;
}
```